katas/helloworld/jorge/Program.cs

```diff
@@ -0,0 +1,2 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Hello, World!");
```

Comment:
Hello @Luisjor , thanks for the latest update, could you please move the file under `katas/main/src/helloworld/jorge`? this is in order to follow the new folder structure we have